### PR TITLE
Add blog post on internship at ePages

### PR DESCRIPTION
**Pull request by cgebken**
This time w/o fail :octocat:

### CODE REVIEW

File: _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+## The company events
+
+Communication and transparency is really important here at ePages.
+To be always up-to-date, we have regular team meetings which gave me the possibility to get an overview of what is going on and who's working at which topic.
+
+Besides these weekly meetings we have some "special events" such as the [ePages Hackathon](https://developer.epages.com/blog/2017/03/29/epages-hackathon-2017.html) or the ePages YOU.
+On our annual Hackathon event in March I got to know many other ePagees from Barcelona, New York, and Jena.
+As it was my first Hackathon, it was a really great experience for me and it was nice to meet a lot of new people.
+
+##  The tasks
```
Comment (roseritter) [inline _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md]: one blank space too much: ## The tasks

File: _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+It is fun to work with so many different nationalities and communicate in different languages.
+At lunch time we eat together, enjoy the sun in the park (yes, there are sunny days in Hamburg!{% emoji wink %}) or grab a coffee.
+This way I've been already shown many nice and delicious restaurants (my favorite one is a Chinese one) and got some insider tips for my first days in Hamburg.
+
+## The company events
+
+Communication and transparency is really important here at ePages.
+To be always up-to-date, we have regular team meetings which gave me the possibility to get an overview of what is going on and who's working at which topic.
+
+Besides these weekly meetings we have some "special events" such as the [ePages Hackathon](https://developer.epages.com/blog/2017/03/29/epages-hackathon-2017.html) or the ePages YOU.
```
Comment (roseritter) [inline _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md]: maybe a short explanation what ePages YOU is?

File: _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+## The secret recipe
+
+It is difficult to determine THE one reason why ePages made me feel welcome so fast.
+Let me present my top three:
+
+1. **Great colleagues** lead to a great working atmosphere.
+2. A structured and detailed [**onboarding**](https://developer.epages.com/blog/2016/08/04/5-steps-to-make-you-feel-comfortable-in-your-new-job.html) simplifies the start tremendously.
+3. Undertaking **responsible tasks** makes you feel like part of the team.
+
+After so many positive aspects, it won't surprise you that I decided to write my master thesis here at ePages after I finished my internship.
```
Comment (roseritter) [inline _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md]: ...after I will finish my internship. 

The internship has not ended yet, has it?

File: _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md (hunk 2 above)
Comment (cgebken) [inline _posts/2017/2017-04-27-how-epages-made-me-feel-welcome-right-away.md]: https://blog.epages.com/de/epages-you/

**Comment by choffmeister** [comment]
🐱  + 🐙  = :octocat:

**Pull request merged**